src/ui/components/ToggleOnMobileCard/index.js

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+
+export class ToggleOnMobileCardBase extends React.Component {
```

Comment:
This name seems odd to me for what it does. How about just calling it `ExpandableCard` ? Either way, it would be nice to see a comment at the top explaining how it's a card that is collapsed by default on mobile.

Reply:
Yeah, it's weird to name. It actually can't be expanded at all on desktop (there's no point), it's just expandable on smaller screens. But I think `ExpandableCard` is still a better name, agreed 👍